content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -132,7 +132,9 @@ stages:
       - performance.json
   training:
     desc: train your model
-    cmd: python train.py
+    cmd:
+      - test -z "$(git status -s)"
+      - python train.py
```

Comment:
Thanks for the update! BTW quick Q. Will the old format also work still? (single command string, not an array)

Reply:
yes it does

Reply:
And would a list with a single item also work the same as the old single-value format?

Reply:
yes as well

Reply:
Got it 👍